sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>This is the implementation of the Azure Client Pipeline</Description>
     <AssemblyTitle>Microsoft Azure Client Pipeline</AssemblyTitle>
-    <Version>1.0.0</Version>
+    <Version>1.0.1</Version>
```

Comment:
This was bumped to 1.0.0-preview.1 3 days ago and now it's 1.0.1 and PR has no description. What exactly is our plan in regards to versioning?

Reply:
@pakrym this is the servicing branch, we are trying to go ahead bump the patch version for everything here in preparation for doing a set of roll-up fixes for Dec. For master the version was bumped to `1.1.0-preview.1` which is still the case.